stockfish/models.py

```diff
@@ -33,7 +34,11 @@ def __init__(
             "UCI_Elo": 1350,
         }
         self.stockfish = subprocess.Popen(
-            path, universal_newlines=True, stdin=subprocess.PIPE, stdout=subprocess.PIPE
+            path,
+            universal_newlines=True,
+            stdin=subprocess.PIPE,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT,
```

Comment:
This is the only solution i could find that doesnt involve running different threads for each pipe. This should be fine as currently the bench command is the only thing stockfish outputs to stderr that is in this python wrapper.